dabest/_api.py

```diff
@@ -4,7 +4,7 @@
 # Email : joseshowh@gmail.com
 
 
-def load(data, idx, x=None, y=None, paired=False, id_col=None,
+def load(data, idx, x=None, y=None, paired=False, preceding=False,  id_col=None,
```

Comment:
Please change `preceding` to `repeated_measures`

Reply:
There are two things going on here: the plot type which is repeated measures, and the comparison type which can be either calculated by substracting the baseline or by subtracting the preceding values in the timeline. Another name for this would be a "sequential" effect size, but in either case it's a new parameter for which type of effect size to calculate.

Reply:
OK, what would help us then is a minimal code example: python code and expected output.

Currently there is no new "plot type" as this is just a refactor of the current shared control plot. 

So it goes back to: we are not just designing plots here but a system to articulate it :) 

I think refactoring the shared control is a good path forward, but then you'd need the user to be clear on what they want: hub-and-spoke (is this different from repeated measure baseline subtracted?) or repeated measure sequential?

Reply:
Yes, baseline-subtracted repeated measures is almost the same as hub-and-spoke, with the critical difference of the pairings. 

I'm wondering it would be simplest to have a plot type with three states:

1. hub-and-spoke "shared_control"
2. repeated measures baseline "rm_baseline"
3. repeated measures sequential "rm_sequential"

Reply:
Hi Prof Claridge and Joses,
Thanks for all the comments and clarifications! I am not very familiar with experiment design and hence I am not too sure about the expected situation of using the plots. However, your comments do enlighten me. May I make some clarifications on the type of plots we have/want to design?

1. unpaired, and not sequential ("shared_control" type) 
2. paired and not sequential ("rm_baseline")
3. paired and sequential ("rm_sequential")

Perhaps my current design is confusing to users because now they would declare 2 parameters in order to plot a graph. Prof Claridge's suggestion to improve on this is great: adding a new parameter `plot_type` with 3 states: "shared_control", "rm_baselin" and "rm_sequential". So I think I shall attempt to remove "paired" and "preceding" but use "plot_type" instead. I will also think about test cases on these graphs to check the correctness of code as well as demonstrate the usage of the functions. 
Once again, thank you for your help! 

Yixuan

Reply:
I think this makes sense, but I'd like to hear from Joses if this makes sense from the overall way DABEST is suitable.